src/components/DatePicker/index.js

```diff
@@ -249,6 +221,8 @@ DatePicker.defaultProps = {
     className: undefined,
     style: undefined,
     locale: undefined,
+    selectionType: 'single',
+    variant: 'single',
 };
 
 export default withReduxForm(DatePicker);
```

Comment:
We already made a global hook (useReduxForm) for ReduxForm integration, check in FileSelector

Reply:
useReduxForm is causing focusable test to fail

Reply:
i think the jestMatcher for toBeDocusable needs modifications in order to work with functional components created with React.forwardRef